website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+
+![Digital Product Passport data model](DigitalIdentityAnchor.svg)
+
+### Core Vocabulary Documentation
+
+The [UNTP core types vocabulary](https://jargon.sh/user/unece/untp-core/v/0.5.0/artefacts/readme/render) defines the uniquely identified Linked Data entities such as Product, Location, Facility, Party, Standard, Regulation, Criteria, Declaration, Attestation, Endorsement. The Digital Identity Anchor only re-uses the UNTP core `VerifiableCredential` and `Identifier` classes.
+
+
+### DIA Documentation
+
+The [DIA documentation](https://jargon.sh/user/unece/DigitalIdentityAnchor/v/0.1.0/artefacts/readme/render) provides a technology-neutral definition of classes, properties and code lists in the DIA model.
+
+## Implementation Guidance
+
+This section provides sample JSON-LD snippets for each DIA component.
+
+### Digital Identity Anchor
+
+The Digital Identity Anchors is a Verifiable credential. Please refer to [DPP VC Guidance](DigitalProductPassport.md#verifiable-credential) for information about the use of the verifiable credentials data model for UNTP.  THe issuing party for the VC MUST be the authority that owns or operates the identity register. 
+
+### Registered Identity
+
+The registered identity class represents the registry member. For example, in a national business register, the registered identity would be one of the registered businesses.
+
+* `type` MUST contain constant value array `["RegisteredIdentity", "Identifier"]` which declares to linked data processors that this is a registered identity which is a sub-type of identity.
+* the `id` MUST contain the globally unique member identifier as maintained in the register.  Often this will be a web link to the member entry.
+* `name` is the human readable registered entity name and SHOULD match the registered name as it appears in the authoritative register.
+* `registeredId` contains the simple identifier value that is unique within the register (but may not be globally unique) for example the VAT registration number.
+* `idScheme` identifies the authoritative register.  If the identity scheme is registered with UN/CEFACT then the `idScheme.id` MUST match the `identityRegister.id` in the UN/CEFACT scheme register.
+* `registerType` is a coded value that allows verifiers to distinguish between different DIA [use cases](#use-cases)
+* `verifiedDIDList` contains an array of DIDs that the member has requested (and the registrar has verified) to be linked to the registeredId of the member.
+* `registrationScopeList` contains a list of URIs that define the scope of the member registration. The values are very specific to the register. For example a national business register would typically have a controlled vocabulary of entity types (eg [Australian Entity Types](https://abr.business.gov.au/Help/EntityTypeList))
+
+
+```json
+  "credentialSubject": {
+    "type": [
+      "RegisteredIdentity",
+      "Identifier"
+    ],
+    "id": "https://sample-register.gov/90664869327",
```

Comment:
I suggest we switch it around. Subject `id` is the DID. This is what verifiers are interested in, so let's not add "proprietary" paths to get to it. 
Add whatever registration URLs in an array. That part is less essential.

Reply:
+1 to this

Reply:
agreed - will update and republish the model